### ISSUE TITLE
custom attr and event not work

### ISSUE THREAD

**Issue by falstack** [SortableJS/vue.draggable.next#53]
```html
<draggable
  :style="computedStyleObject"
  @click="handleClickEvent"
>
  ...
</draggable>
```

`style` attribute and `click` event not work, but vue2 version can work, it can be resolve😭?

**Comment by falstack** [comment]
@David-Desmaisons

https://github.com/SortableJS/vue.draggable.next/pull/55

**Comment by David-Desmaisons** [comment]
Did you try ` @click`.native="handleClickEvent"` ?

**Comment by falstack** [comment]
@David-Desmaisons vue3 not support`native`

**Comment by falstack** [comment]
@David-Desmaisons  plz add this line：[see](https://github.com/SortableJS/vue.draggable.next/pull/55/commits/214aba57b5436b587570087aff1447560a7ed24b#diff-05c17c0a5abd2deb685a7a1a722989eb8b13fbfccf0ad32ad7bfbd43986fe3b5R133)

**Comment by David-Desmaisons** [comment]
@falstack I will review and merge this week-end.

**Comment by David-Desmaisons** [comment]
@falstack  Corrected in version 4.1.0
